packages/sui-js/src/pipe/index.js

```diff
@@ -0,0 +1,4 @@
+const pipe = (...funcs) => arg =>
+  funcs.reduce((value, func) => func(value), arg)
```

Comment:
it would be nice if you could add some tests

Reply:
I would rename `(value, func)` into `(acc, value)` for easier reading as so are called the args passed through the reducer fn, up to you

Reply:
ok thx!!